src/OpenRiaServices.Server.EntityFrameworkCore/Test/DbContextModel/EFCoreModels.csproj

```
@@ -5,9 +5,9 @@
     <Version>1.0.0.0</Version>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.24" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.24" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.24" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.*" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.*" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.*" />
```

Should this be changed to .32 to match version in web.config ?

```suggestion
    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.32" />
```